### PR TITLE
Remove refCols from mapper policies.

### PR DESCRIPTION
**Pull request by ktlim**
Also clean up some spurious calibDates.

### CODE REVIEW

File: policy/lsstCamMapper.yaml [hunk 1]
```diff
@@ -79,12 +79,11 @@ calibrations:
   qe_curve:
     columns:
     - detector
-    - calibDate
+    - dateObs
```
Comment (SimonKrughoff) [inline policy/lsstCamMapper.yaml]: I was under the impression these were the columns in the calib repository to find the calib in question.  I must be wrong about that, but wanted to bring it up.

Comment (ktlim) [reply policy/lsstCamMapper.yaml]: These are columns in the _main_ registry that are used to get values to look up in the calib registry.  The `obsTimeName` entry below is the name of the field whose value is compared against the `(validStart, validEnd)` interval.

Comment (ktlim) [reply policy/lsstCamMapper.yaml]: This is unfortunately not well-documented, but there is a code comment to this effect at https://github.com/lsst/obs_base/blob/master/python/lsst/obs/base/mapping.py#L490-L491

Comment (SimonKrughoff) [reply policy/lsstCamMapper.yaml]: OK.  Thanks for the clarification.

**Pull request merged**